examples/dynamo/eg3/solver_mod.x90

```diff
@@ -437,7 +437,7 @@ contains
   call invoke_set_field_scalar(1.0_r_def, Ax)
   call invoke( matrix_vector_kernel_mm_type(lumped_weight,Ax,mm) )
 
-  !PSY call invoke ( divide_field(rhs, lumped_weight, lhs))
+  !PSY call invoke ( divide_fields(rhs, lumped_weight, lhs))
```

Comment:
Added "! Modified by I Kavcic, Met Office." as in eg4/solver_mod.f90 file (below).